component/build.gradle

```diff
@@ -32,6 +32,8 @@ dependencies {
     implementation 'org.springframework.amqp:spring-amqp'
     implementation 'org.springframework.amqp:spring-rabbit'
 
+    implementation 'com.synopsys.integration:blackduck-common', rootProject.ext.blackduckCommonExcludes
```

Comment:
`rootProject.ext.blackduckCommonExcludes` was removed in a PR at the end of last week. The exclude is now applied to all subprojects automatically. This should be removed.